docs/install/quick-start.rst

```diff
@@ -0,0 +1,36 @@
+.. meta::
+  :description: Install rocDecode
+  :keywords: install, rocDecode, AMD, ROCm
+
+
+Installation
+*********************************
+
+
+Prerequisites
+===============
+
+* gfortran version 7.5.0 or newer
```

Comment:
Should this be GFortran

Perhaps it should have a link, too:
https://fortran-lang.org/en/learn/os_setup/install_gfortran/

Reply:
Fixed.